src/import/instruments.ts

```diff
@@ -38,12 +38,14 @@ interface PastedTimeProfileRow {
   Weight?: string
   'Source Path'?: string
   'Symbol Name'?: string
+  'Symbol Names'?: string
 }
 
 interface PastedAllocationsProfileRow {
   'Bytes Used'?: string
   'Source Path'?: string
   'Symbol Name'?: string
+  'Symbol Names'?: string
```

Comment:
This is still `Symbol Name` for allocations but if we don't add this field here we have a unknown field. Couldn't figure out nicer way of dealing with this